Olaheavy2021/registrationApp
# Student Dashboard & Profile Page: Properly styled page created and structured

# Ticket Number and description
[CBP-19: Student Dashboard & Profile Page: Properly styled page created and structured](https://sunkanmiolawuwogas.atlassian.net/browse/CBP-19)

## Context
As a student, I want a visually appealing homepage and profile page for the web application, So I can easily navigate and access the list of modules and contains links to other student pages.
## Work Done

Create a Student Home page and Profile page for the web application using Bootstrap, CSS and HTML.

## Screenshots
<img width="701" alt="student_dashboard" src="https://github.com/Olaheavy2021/registrationApp/assets/86934237/eb985f51-2f59-475e-8d60-f0715436d3ea">
<img width="742" alt="edit_profile" src="https://github.com/Olaheavy2021/registrationApp/assets/86934237/f2e30c20-5549-4f84-90e1-80707e8b887c">
<img width="710" alt="reset_password" src="https://github.com/Olaheavy2021/registrationApp/assets/86934237/74f1f1fc-66f1-4389-b17a-d2f1f4103f32">

## Testing instructions
- Navigate to the Login Page
- Sign with the required credentials
- The user should be redirected to the Student Dashboard Page
## Acceptance Criteria

- [x] The pages have an attractive layout and visually pleasing elements.
- [x] The pages are divided into at least four content sections.
- [x] The pages are responsive, ensuring compatibility across different devices and screen sizes.
